fe/fe-core/src/main/java/com/starrocks/qe/StmtExecutor.java

```diff
@@ -819,13 +819,21 @@ private void handleAnalyzeStmt() throws IOException {
                     new HistogramStatisticsCollectJob(db, table, analyzeStmt.getColumnNames(),
                             StatsConstants.AnalyzeType.HISTOGRAM, StatsConstants.ScheduleType.ONCE,
                             analyzeStmt.getProperties()));
+
+            //Could populate column statistic cache after Analyze table manually
+            GlobalStateMgr.getCurrentStatisticStorage().expireHistogramStatistics(table.getId(), analyzeStatus.getColumns());
+            GlobalStateMgr.getCurrentStatisticStorage().getHistogramStatistics(table, analyzeStatus.getColumns());
```

Comment:
Is it means re-creating after expired? Maybe better to put them inside of refreshXXX like `GlobalStateMgr.getCurrentAnalyzeMgr().refreshBasicStatisticsCache`